src/AbpCompanyName.AbpProjectName.Application/AbpCompanyName.AbpProjectName.Application.csproj

```diff
@@ -111,6 +109,9 @@
       <HintPath>..\packages\System.Linq.Dynamic.1.0.7\lib\net40\System.Linq.Dynamic.dll</HintPath>

```

Comment:
Remove System.Linq.Dynamic packages from all projects since System.Linq.Dynamic.Core has the same functionality with .netstandard support.